smart_open/tests/test_ssh.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import mock
+import unittest
+
+import smart_open.ssh
+
+
+def mock_ssh(func):
+    def wrapper(*args, **kwargs):
+        smart_open.ssh._SSH.clear()
+        return func(*args, **kwargs)
```

Comment:
A newline would help readability here.

```suggestion
        return func(*args, **kwargs)

```